gcp_flowlogs_reader/gcp_flowlogs_reader.py

```diff
@@ -229,7 +229,7 @@ def _reader(self):
                 iterator = self.logging_client.list_entries(
                     filter_=expression,
                     page_size=self.page_size,
-                    projects=[project],  # only collects current project flows
+                    resource_names=[project],  # only current project flows
```

Comment:
Little concerned about this; I haven't yet convinced myself that `projects=` and `resource_names=` are identical.

Reply:
Based on [this](https://googleapis.dev/python/logging/latest/UPGRADING.html#support-for-non-project-resources) I think i'ts supposed to be:

```suggestion
                    resource_names=[f'projects/{project}'],  # only current project flows
```

Reply:
Tested this with your branch; that seems to do the trick.